src/modules/map/component.ts

```diff
@@ -92,6 +92,23 @@ export async function createMapComponent(components: {
     return oldTokens
   }
 
+  function expireOrders(tiles: Record<string, Tile>) {
+    const newTiles: Record<string, Tile> = {}
+
+    for (const id in tiles) {
+      const tile = tiles[id]
+      const notExpired = !isExpired(tile)
+
+      newTiles[id] = {
+        ...tile,
+        price: notExpired ? tile.price : undefined,
+        expiresAt: notExpired ? tile.expiresAt : undefined
```

Comment:
could we change the above to:

```ts
const tile = tiles[id]
if (isExpired(tile)) {
  newTiles[id] = { ...tile }
  delete newTiles[id].price
  delete newTiles[id].expiresAt
} else {
  newTiles[id] = tile
}
```

This is so the expired tiles are stored in memory exactly the same as tiles that are not on sale. Also this way we only create a new object if necessary (if the tile is expired), otherwise we keep the old object.